bgpd/bgp_attr.c

```diff
@@ -863,6 +863,11 @@ unsigned int attrhash_key_make(const void *p)
 	key = jhash(attr->mp_nexthop_local.s6_addr, IPV6_MAX_BYTELEN, key);
 	MIX3(attr->nh_ifindex, attr->nh_lla_ifindex, attr->distance);
 	MIX3(attr->bh_type, attr->otc, bgp_attr_get_aigp_metric(attr));
+	MIX(attr->mm_seqnum);
```

Comment:
Please use MIX3 to bundle df_alg, df_pref, mm_seqnum.

Reply:
Done.